scalabel/eval/mot.py

```diff
@@ -339,9 +322,9 @@ def generate_results(
 
 
 def evaluate_track(
-    acc_single_video: VidFunc[Video],
-    gts: List[Video],
-    results: List[Video],
+    acc_single_video: VidFunc,
+    gts: List[List[Frame]],
+    results: List[List[Frame]],
```

Comment:
so the support for inputting paths instead of the actual frames is removed? If so, the function documentation should be updated too.